2009-01-28  Ulrich Drepper  <drepper@redhat.com>

	* nss/getent.c (print_networks): Don't print comma between aliases.

--- libc/nss/getent.c	2 Jan 2009 08:46:57 -0000	1.32
+++ libc/nss/getent.c	28 Jan 2009 15:55:04 -0000	1.33
@@ -448,8 +448,6 @@ print_networks (struct netent *net)
       putchar_unlocked (' ');
       fputs_unlocked (net->n_aliases[i], stdout);
       ++i;
-      if (net->n_aliases[i] != NULL)
-	putchar_unlocked (',');
     }
   putchar_unlocked ('\n');
 }
